verifiable/onchain/credential.go

```diff
@@ -0,0 +1,61 @@
+package onchain
+
+type CredentialSubjectField struct {
```

Comment:
give comments public  to structures , not sure why linter didn't fail

Reply:
Done